packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -694,7 +694,8 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
   private _onKeyDown = (ev: React.KeyboardEvent<HTMLElement>) => {
     if (ev.which === KeyCodes.escape ||
-      ev.altKey ||
+      ev.which === KeyCodes.alt ||
+      (ev.which === KeyCodes.up && ev.altKey) ||
```

Comment:
In my testing contextual menus already collapse on ALT + UpArrow... Are changes to this file needed?

Reply:
You are correct. I went and confirmed that. I'll remove this, but it looks like the changes for the combo box are still needed.